Makefile

```diff
@@ -36,7 +40,7 @@ build: lint $(BINDIR)/$(BINNAME)
 SRC := $(shell find . -type f -name '*.go' -print) go.mod go.sum
 
 $(BINDIR)/$(BINNAME): $(SRC)
-	GO111MODULE=on go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./cmd/hypper
```

Comment:
I suppose removing `GO111MODULE=on` is ok :thinking: . Afterall, we are only building and testing on 1.14, 1.15, and it is default since 1.13.

Reply:
Yes, also @mattfarina mentioned that we do not need it any more.